dts/arm/st/mp1/stm32mp157.dtsi

```diff
@@ -147,6 +147,56 @@
 			};
 		};
 
+		spi1: spi@44004000 {
```

Comment:
Split in dedicated commit, along with fixup changes

Reply:
Splitted, I also added changes related to STM32MP1x SoC, so commit are split in soc support, driver support and board support